packages/connect-react/src/index.css

```diff
@@ -320,3 +333,228 @@ button.cb-secondary-button {
   width: 100%;
   margin: 1rem 0rem;
 }
+
+.cb-passkey-list-container {
```

Comment:
Just one comment on the .css file: IMO in the future we should consider either splitting it up into multiple files or introducing multiple sections (one for login, append, passkey-list)